src/routes/edit/share-options/exports/frameworks/angular-fragment.ts

```diff
@@ -150,10 +150,10 @@ const getComponentCode = (fragment: any, fragments: any[]) => {
 	// component.html
 	componentCode[`src/app/components/${tagNameFromFragment(fragment)}/${tagNameFromFragment(fragment)}.component.html`] =
 		format(jsonToTemplate(fragment.data, fragments), formatOptionsHtml);
-
 	// module.ts
 	componentCode[`src/app/components/${tagNameFromFragment(fragment)}/${tagNameFromFragment(fragment)}.module.ts`] = format(
 		`import { NgModule } from "@angular/core";
+		import { CommonModule } from "@angular/common";
```

Comment:
This seems like an overkill. Why wouldn't this be in the list of imports for the component?

Reply:
Oh, because we're importing from angular/common module not carbon-components-angular. is there a way we can specify to import from angular/common inside a component?

Reply:
React export has a notion of `otherImports`.

We didn't need it before in angular. Now that we do, it should be straightforward to add it in a similar way.